src/cli/ui.py

```diff
@@ -70,7 +69,8 @@ def handle_cmd(self, options=[]):
                 title()
                 data = option[2]()
 
-                print(data if data else "Successfully executed.\n")
+                print(data if data else
```

Comment:
Are we checking here if `data` is an empty string? If so, I believe it's sufficient to just do: `data or <expr>`